pkg/crc/cache/cache.go

```diff
@@ -32,6 +32,10 @@ func NewPodmanCache(destDir string) *Cache {
 	return New(constants.PodmanBinaryName, constants.GetPodmanUrl(), destDir)
 }
 
+func NewGoodhostCache(destDir string) *Cache {
+	return New(constants.GoodhostBinaryName, constants.GetGoodHostUrl(), destDir)
```

Comment:
`Goodhost` or `GoodHost`? And this really should be `Goodhosts` or `GoodHosts`  (applies throughout the PR)

Reply:
@cfergeau I am going to use `Goodhost` throughout the PR.

Reply:
`git grep  goodhost[^s]` still returns some results